src/cmdstan/arguments/arg_laplace_jacobian.hpp

```diff
@@ -0,0 +1,23 @@
+#ifndef CMDSTAN_ARGUMENTS_ARG_LAPLACE_JACOBIAN_HPP
+#define CMDSTAN_ARGUMENTS_ARG_LAPLACE_JACOBIAN_HPP
+
+#include <cmdstan/arguments/singleton_argument.hpp>
+
+namespace cmdstan {
+
+class arg_laplace_jacobian : public bool_argument {
+ public:
+  arg_laplace_jacobian() : bool_argument() {
+    _name = "jacobian";
+    _description = "Jacobian adjustment?";
```

Comment:
this can just be "set to true to include change-of-variables adjustment for constraining parameter transforms"

I don't think we want to go into True corresponding to max a posteriori (MAP) and False corresponding to a (penalized) maximum likelihood estimate (MLE).

Reply:
fixed.